airflow/contrib/utils/gcp_field_validator.py

```diff
@@ -134,6 +134,7 @@
 
 import re
 
+from typing import Function
```

Comment:
Hi @jmcarp , thanks for picking up this Flake8 issue quickly.

May you fix the errors caused by this line?